api/candlepin-api-spec.yaml

```diff
@@ -30,6 +30,336 @@ paths:
         default:
           $ref: '#/components/responses/default'
 
+  /activation_keys:
+    get:
+      description: Retrieves a list of activation keys
+      tags:
+        - Activation Key
+      operationId: findActivationKey
+      x-java-response:
+        type: Iterable
+        isContainer: true
+      security: []
+      responses:
+        200:
+          description: List of activation keys
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ActivationKeyDTO'
+        default:
+          $ref: '#/components/responses/default'
+
+  /activation_keys/{activation_key_id}:
+    get:
+      description: Retrieves a single activation key
+      tags:
+        - Activation Key
+      operationId: getActivationKey
+      security: []
+      parameters:
+        - name: activation_key_id
+          in: path
+          description: The Id of activation key to retrieve
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Activation key
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ActivationKeyDTO'
+        400:
+          description: Activation key Id is null or empty or could not be found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: Activation key Id is null or empty or could not be found
```

Comment:
There should be a comma somewhere in that list at the end. Any of the following would work, depending on your preference for the Oxford comma, and the intended coupling of "null" and "empty":

Activation key ID is null, empty, or could not be found
Activation key ID is null, empty or could not be found
Activation key ID is null or empty, or could not be found

Reply:
Done.